app/views/home/index.slim

```diff
@@ -1,3 +1,4 @@
+//Trending posts
```

Comment:
We can avoid the `//` comments here. We should only have comments for 'why' the code is doing something, not 'what' the code is doing (ie. it should be self-explanatory).